test/AElf.Contracts.SolidityContract.Tests/BallotContractTests.cs

```diff
@@ -39,7 +39,7 @@ public async Task<Address> DeployBallotContractTest()
     public async Task ReadProposalsTest()
     {
         var contractAddress = await DeployBallotContractTest();
-        var tx = GetTransaction(DefaultSenderKeyPair, contractAddress, "proposals");
+        var tx = await GetTransactionAsync(DefaultSenderKeyPair, contractAddress, "proposals");
```

Comment:
minor thought: for `GetTransactionAsync` do u foresee it returning errors in the form of exceptions in future? or do u think tx==null is a sufficient check for issues?